components/header/Header.tsx

```diff
@@ -17,17 +17,36 @@ export interface Props {
    * @title Navigation items
    * @description Navigation items used both on mobile and desktop menus
    */
-  navItems?: SiteNavigationElement[] | null;
+  navItems?: NavItemProps[];
 
   /** @title Logo */
   logo?: { src: ImageWidget; alt: string };
+
+  /** 
+* @title Menu Item Blog
+*/
+  blogItem: { text: string; href: string };
+
+  /** 
+  * @title Menu Item Help
+  */
+  helpItem: { text: string; href: string };
+
+  /** 
+* @title Flag Icon
+* @Description Internationalization Menu Flag Icon
+*/
+  countryFlag: { countryLabel: string, countryImg: { img: ImageWidget, alt: string } }[]
```

Comment:
Prefira criar tipagens especificas para poder reutilizar entre os arquivos com export/import
```suggestion
  countryFlag: Country[]
```